shared/devices/device-page/dumb.native.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import Render from './index.render.native'
+
+const common = {
+  type: 'desktop',
+  name: 'Home Computer',
+  isCurrent: false,
+  isRevoked: false,
+  timeline: null,
+  banner: null,
+  onRevoke: () => {}
+}
+
+export default {
+  'Device Page': {
```

Comment:
🐑 Would be nice to use the type https://github.com/keybase/client/blob/master/shared/constants/types/more.js#L24

Example: https://github.com/keybase/client/blob/master/shared/common-adapters/dumb.native.js#L8